tutor/src/screens/assignment-review/ux.js

```diff
@@ -211,6 +211,10 @@ export default class AssignmentReviewUX {
     );
   }
 
+  @computed get isDroppedQuestionsSaving() {
+    return this.taskPlan.api.isPending;
+  }
```

Comment:
FYI `api.isPending` will also be true for any network activity like `get`.  To check for put you might add a isUpdating or something like that to api that checks `requestsInProgress` like `fetchInProgress` does: https://github.com/openstax/tutor-js/blob/wrm/shared/src/model/api.js#L30-L32

Reply:
Ah good note! I like that idea, I should add that while checking the other modals for async...